spark-cassandra-connector/src/it/scala/com/datastax/spark/connector/streaming/ActorStreamSpec.scala

```diff
@@ -23,6 +24,12 @@ class ActorStreamingSpec extends ActorSpec with CounterFixture with ImplicitSend
     session.execute("TRUNCATE demo.streaming_wordcount")
   }
 
+  override def afterAll(configMap: ConfigMap) {
```

Comment:
I wonder whether it should be in `ActorSpec` and close streaming context as well: `ssc.stop(stopSparkContext=true)`, @helena wdyt?